AntennaTracker/servos.cpp

```diff
@@ -130,12 +130,9 @@ void Tracker::update_pitch_onoff_servo(float pitch)
 */
 void Tracker::update_pitch_cr_servo(float pitch)
 {
-    int32_t pitch_min_cd = g.pitch_min*100;
-    int32_t pitch_max_cd = g.pitch_max*100;
-    if ((pitch>pitch_min_cd) && (pitch<pitch_max_cd)) {
-        g.pidPitch2Srv.set_input_filter_all(nav_status.angle_error_pitch);
-        SRV_Channels::set_output_scaled(SRV_Channel::k_tracker_pitch, g.pidPitch2Srv.get_pid());
-    }
+    g.pidPitch2Srv.set_input_filter_all(nav_status.angle_error_pitch);
+    const float pitch_out = constrain_float(g.pidPitch2Srv.get_pid(), -(-g.pitch_min+g.pitch_max) * 100/2, (-g.pitch_min+g.pitch_max) * 100/2);
```

Comment:
I'm having trouble understanding this constraint.  Why would be want to constrain to any sort of average of min and max?

Reply:
its what the range is setup as 
https://github.com/ArduPilot/ardupilot/blob/36b2d76cc0655a07da8ae7903c5eb06ffa2c1d8f/AntennaTracker/servos.cpp#L20